src/components/Modal.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+import React from "react";
+import Image from "next/image";
+import nought from "@/icons/icon-o.svg";
+import Link from "next/link";
+
+interface ModalProps {
+  gameResult?: string;
+  firstBtnLabel: string;
+  secondBtnLabel: string;
+  showModal: boolean;
+  navigate: string;
+  onFirstBtnClick: () => void;
+  onSecondBtnClick: () => void;
+}
+
+export default function Modal({
+  gameResult,
+  firstBtnLabel,
+  secondBtnLabel,
+  showModal,
+  navigate,
```

Comment:
This name is not a good name. There are 2 buttons here. It's not clear what button navigates